package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/browserslist-config-tao",
-    "version": "0.3.0",
+    "version": "0.4.0",
```

Comment:
you don't need to bump version anymore and it will be a breaking change (so for example apps that still need IE 11 won't update to the new version)